commit 0b147469f2cac30f43a5fa9b7a2d15a27a461360
Author: Florian Meier <florian.meier@koalo.de>
Date:   Mon Dec 2 20:28:22 2013 +0100

    BCM2708: Add I2S and DMA support to default config
    
    This commit adds several modules that are needed for
    I2S support for the Raspberry Pi to the defconfig.
    
    Signed-off-by: Florian Meier <florian.meier@koalo.de>

diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index 0c58172..38ccd22 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -741,6 +741,13 @@ CONFIG_SND_USB_UA101=m
 CONFIG_SND_USB_CAIAQ=m
 CONFIG_SND_USB_CAIAQ_INPUT=y
 CONFIG_SND_USB_6FIRE=m
+CONFIG_SND_SOC=m
+CONFIG_SND_SOC_DMAENGINE_PCM=y
+CONFIG_SND_SOC_GENERIC_DMAENGINE_PCM=y
+CONFIG_SND_BCM2708_SOC_I2S=m
+CONFIG_SND_BCM2708_SOC_HIFIBERRY_DAC=m
+CONFIG_SND_SOC_I2C_AND_SPI=m
+CONFIG_SND_SOC_PCM5102A=m
 CONFIG_SOUND_PRIME=m
 CONFIG_HIDRAW=y
 CONFIG_HID_A4TECH=m
@@ -931,6 +938,10 @@ CONFIG_RTC_DRV_RS5C348=m
 CONFIG_RTC_DRV_DS3234=m
 CONFIG_RTC_DRV_PCF2123=m
 CONFIG_RTC_DRV_RX4581=m
+CONFIG_DMADEVICES=y
+CONFIG_DMA_BCM2708=m
+CONFIG_DMA_ENGINE=y
+CONFIG_DMA_VIRTUAL_CHANNELS=m
 CONFIG_UIO=m
 CONFIG_UIO_PDRV_GENIRQ=m
 CONFIG_STAGING=y
